README.md

```diff
@@ -1,79 +1,142 @@
-# nginx-k8s-loadbalancer
+<div>
+    <span>
+<img src="nkl-logo.svg" width="124" />
+</span>    
+<span>
+<h2>nginx-k8s-loadbalancer</h2>
+</span>
+</div>
 
-## Welcome to the Nginx Kubernetes Load Balancer project !
+The NGINX K8s Loadbalancer, or _NKL_, is a Kubernetes controller that provides TCP load balancing external to a Kubernetes cluster running on-premise.
 
-<br/>
+## Requirements
```

Comment:
I think we should mention:
* NGINX Plus
* Kubernetes running on prem
* Any other things I'd need? Specific architectures? OS?

Reply:
✅